components/Projects.js

```diff
@@ -9,41 +9,39 @@ export default function Projects() {
           Projects
         </h1>
       </div>
-      {/* Grid starts here */}
-      <div className="bg-[#F1F1F1] dark:bg-gray-900">
-        <div className="max-w-6xl mx-auto grid grid-cols-1 md:grid-cols-2 gap-8 py-20 pb-40">
-          {userData.projects.map((proj, idx) => (
-            <ProjectCard
-              title={proj.title}
-              link={proj.link}
-              imgUrl={proj.imgUrl}
-              number={`${idx + 1}`}
-            />
-          ))}
-        </div>
+      <div className="bg-[#F1F1F1] dark:bg-gray-900 p-10 grid grid-cols-1 md:grid-cols-2">
+        {userData.projects.map((project, index) => (
+          <ProjectCard projectID={index} />
+        ))}
       </div>
     </section>
   );
 }
 
-const ProjectCard = ({ title, link, imgUrl, number }) => {
+const ProjectCard = ({ projectID }) => {
+  const project = userData.projects[projectID];
+
   return (
-    <a href={link} className="w-full block shadow-2xl">
-      <div className="relative overflow-hidden">
-        <div className="h-72 object-cover">
-          <img
-            src={imgUrl}
-            alt="portfolio"
-            className="transform hover:scale-125 transition duration-2000 ease-out object-cover h-full w-full"
-          />
-        </div>
-        <h1 className="absolute top-10 left-10 text-gray-50 font-bold text-xl bg-red-500 rounded-md px-2">
-          {title}
-        </h1>
-        <h1 className="absolute bottom-10 left-10 text-gray-50 font-bold text-xl">
-          {number.length === 1 ? "0" + number : number}
-        </h1>
+    <div className="rounded flex flex-col justify-between transform hover:scale-105 transition duration-200 shadow-md my-4 mx-4">
```

Comment:
I like this kind of emphasis behavior (in this case scrolling over the card slightly scales it up) but it looks a little janky with the text scaling up, open to suggestions for other ways to do this!

Reply:
Nice! I think you have to be pretty careful with scaling - especially with something this big. It's also worth being careful not to mislead a user into thinking something's interactive when it's not (in this case, the card that scales can't be interacted with, which feels a bit strange)

Instead, I'd suggest removing the scaling behavior from the entire card and instead apply it to just the link buttons - see illustrated in my example PR.